nin/frontend/app/scripts/services/ninrc.js

```diff
@@ -0,0 +1,9 @@
+function ninrc($http) {
+  const xhr = new XMLHttpRequest();
```

Comment:
Yeah, it needs to be sync. Perhaps we should have specified that in a source comment.